dhis-2/dhis-api/src/main/java/org/hisp/dhis/preheat/Preheat.java

```diff
@@ -102,6 +107,8 @@
      */
     private Map<Class<?>, Map<String, Map<String, String>>> uniqueAttributeValues = new HashMap<>();
 
+    private Map<Class<?>, List<Attribute>> mapAllClassAttributes = new HashMap<>();
```

Comment:
would be good if the name helps to understand what the `Class` for the keys represents. Something like `attributesByTargetObjectType` (not really sure what it is here).

Reply:
This map contains list attributes of all classes that user are importing. 
Later it could be used by other `ObjectValidationCheck` which need to validate other attribute value type.